### PR TITLE
feat(layout): `Cell` add border-radius

### PR DESCRIPTION
**Pull request by mdlufy**
Close #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/layout/components/cell/cell.styles.less [hunk 1]
```diff
@@ -81,6 +81,10 @@
     }
 
     &[data-size='s'] {
+        &:hover {
```
Comment (vladimirpotekhin) [inline projects/layout/components/cell/cell.styles.less:84]: I think we need to add borders to the cell, not just for hover state.

File: projects/layout/components/cell/cell.styles.less [hunk 2]
```diff
@@ -81,6 +81,10 @@
...
+            border-radius: 0.5rem;
```
Comment (vladimirpotekhin) [inline projects/layout/components/cell/cell.styles.less:85]: probably we can use approach as below with --t-height, --t-padding
like 
`--t-radius: ...`

also please check the proprietary radiuses, maybe they fit the layout

File: projects/layout/components/cell/cell.styles.less [hunk 3]
```diff
@@ -137,6 +141,11 @@
     }
 
     &[data-size='l'] {
+        border-radius: var(--tui-radius-l);
+
+        --tui-radius-s: 0.5rem;
```
Comment (waterplea) [inline projects/layout/components/cell/cell.styles.less:146]: We shouldn't override those variables.

Comment (mdlufy) [reply projects/layout/components/cell/cell.styles.less]: yes, it was for debug ))

File: projects/layout/components/cell/cell.styles.less (hunk 1 above)
Comment (mdlufy) [reply projects/layout/components/cell/cell.styles.less]: fixed

File: projects/layout/components/cell/cell.styles.less (hunk 2 above)
Comment (mdlufy) [reply projects/layout/components/cell/cell.styles.less]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/layout/components/cell/cell.styles.less [hunk 4]
```diff
@@ -3,6 +3,7 @@
 [tuiCell] {
     --tui-height: calc(var(--tui-height-s) - 0.5rem);
```
Comment (waterplea) [inline projects/layout/components/cell/cell.styles.less:4]: Is this used anywhere?

**Review by waterplea — APPROVED** [review]

File: projects/layout/components/cell/cell.styles.less (hunk 4 above)
Comment (mdlufy) [reply projects/layout/components/cell/cell.styles.less:4]: No, it used `--t-height` only
It's misspel and maybe rename `--tui-height` to `--t-height` on host OR just remove `--tui-height` ?

image
image

**Pull request merged**